src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -490,7 +502,7 @@ function showVersionWarningBanner(data) {
   inner.classList = "sidebar-message";
   button.classList =
     "btn text-wrap font-weight-bold ms-3 my-1 align-baseline pst-button-link-to-stable-version";
-  button.href = `${preferredURL}${DOCUMENTATION_OPTIONS.pagename}.html`;
+  button.href = getCurrentUrlPath();
```

Comment:
this isn't right, it still needs to incorporate `preferredURL` otherwise the link will just go to the page you're currently on (AKA an outdated version of the docs, instead of current stable docs)

Reply:
this?

```suggestion
  button.href = `${preferredURL}{getCurrentUrlPath()}`;
```

Reply:
missed the second `$`
```suggestion
  button.href = `${preferredURL}${getCurrentUrlPath()}`;
```